cns/NetworkContainerContract.go

```diff
@@ -90,6 +90,16 @@ type CreateNetworkContainerRequest struct {
 	EndpointPolicies           []NetworkContainerRequestPolicies
 }
 
+// CreateNetworkContainerRequest implements fmt.Stringer for logging
+func (req *CreateNetworkContainerRequest) String() string {
+	return fmt.Sprintf("[Azure CNS] cns.CreateNetworkContainerRequest "+
```

Comment:
this log metadata shouldn't be in the String method, it should only return a string representation of the struct
```suggestion
	return fmt.Sprintf("CreateNetworkContainerRequest"+
```